folly/Traits.h

```diff
@@ -434,10 +434,10 @@ namespace traits_detail_IsNothrowSwappable {
 // enabled.
 template <typename T>
 using IsNothrowSwappable = std::is_nothrow_swappable<T>;
-#elif _CPPLIB_VER
-// MSVC defines the base even if C++17 is disabled, and MSVC has
-// issues with our fallback implementation due to over-eager
-// evaluation of noexcept.
+#elif _CPPLIB_VER > 800
+// MSVC 2015+ defines the base even if C++17 is disabled, and
```

Comment:
MSVC 2015 is not supported any longer and there are various parts of folly that no longer build with that compiler. It does not seem useful to have any references to that version or earlier in the code or in the comments.

The (current) earliest supported MSVC is some update within the 2017 line.

Reply:
fair enough, fixed.